data/json/dreams.json

```diff
@@ -563,6 +602,8 @@
     "type": "dream",
     "messages": [
       "You feel smug, knowing that the Queen is dependent on you.",
+      "You relax and enjoy the humming adulation of all your followers.",
+      "Something dangerous has entered your area!  You send a couple of hundred followers to deal with it.",
```

Comment:
Nitpick but I'm pretty sure no eusocial insect does that, the queen does not really gives order and wouldn't know if an intruder came into the colony.